frontend/src/utils/configs.js

```diff
@@ -2,5 +2,7 @@ export const USER_SVC_BASE_URL =
   window.__RUNTIME_CONFIG__.REACT_APP_URI_USER_SVC || "http://localhost:8000";
 export const MATCH_SVC_BASE_URL =
   window.__RUNTIME_CONFIG__.REACT_APP_URI_MATCH_SVC || "http://localhost:8001";
+export const COMMUNICATION_SVC_BASE_URL =
+  window.__RUNTIME_CONFIG__.REACT_APP_URI_MATCH_SVC || "http://localhost:8005";
```

Comment:
I think should be `window.__RUNTIME_CONFIG__.REACT_APP_URI_COMMUNICATION_SVC` right?

Reply:
fixed.